src/site/content/en/blog/codelab-address-form-best-practices/index.md

```diff
@@ -0,0 +1,285 @@
+      <option value="AS">American Samoa</option>
+  </select>  
+</section>
+```
+
+You'll see that **Postal code** is optional: that's because [many countries don't use postal codes](https://hellowahab.wordpress.com/2011/05/24/list-of-countries-without-postal-codes/). 
+([Global Sourcebook](https://www.grcdi.nl/gsb/global%20sourcebook.html) provides information about 
+address formats for 194 different countries, including sample addresses.) The label **Country or 
+region**  is used instead of **Country**, because some options from the full list (such as the 
+United Kingdom) are not single countries (despite the `autocomplete` value!)
+
+* You've built a highly functional address form, but what if your site requires more than one 
+address, for shipping and billing, for example? Try updating your form to enable customers to enter 
+shipping and billing addresses. How can you make data entry as quick and easy as possible, 
+especially if the two addresses are the same? The article that goes with this codelab explains 
+[techniques for handling multiple addresses](/payment-and-address-form-best-practices#billing-address). 
+Whatever you do, make sure to use the correct `autocomplete` values!
+
+Lastly, add a telephone number input:
+
+```html
+<section>
+  <label for="tel">Telephone</label>
+  <input id="tel" name="tel" autocomplete="tel" type="tel" 
+    maxlength="30" pattern="[\d \-\+]+" enterkeyhint="done" 
+    required>
+</section>
+```
+
+For phone numbers use a single input: don't split the number into parts. That makes it easier for 
+users to enter data or copy and paste, makes validation simpler, and enables browsers to autofill. 
+
+Two new things to be aware of here: 
+* `type="tel"` ensures mobile users get the right keyboard.
+* `enterkeyhint="done"` sets the mobile keyboard enter key label to show that this is the last 
+field and the form can now be submitted (the default is `next`).
+
+<figure class="w-figure">
+  <img class="w-screenshot" src="images/enterkeyhint.png" alt="Two screenshots of a form 
+  on Android showing how the enterkeyhint input attribute changes the enter key button icon.">
+  <figcaption class="w-figcaption">Use the enterkeyhint attribute to set the Enter button label: 'next' and 'done'.</figcaption>
+</figure>
+
+{% Aside 'gotcha' %}
+Using type="number" adds an up/down arrow to increment numbers, which makes no sense for data such 
+as telephone, payment card or account numbers. Instead, you should use `type="tel"` for telephone 
+numbers. For other numbers, use `type="text"` (or leave off the attribute, since `text` is the 
+default) and add `inputmode="numeric"` to get a numeric keyboard on mobile. 
+{% endAside %}
+
+* Can you see any problems with using a single input for telephone number? Do you store phone 
+number parts (country and area code) separately? If so, why?
+
+Your complete address form should now look like this:
+
+{% Glitch {
+  id: 'address-form',
```

Comment:
I'm getting "page not found" for this one https://glitch.com/~address-form

Reply:
Yeah :/... I've filed a bug, and Rob has emailed them. I think it may be a side effect of their fix to the other bug I filed :).